typing/predef.ml

```diff
@@ -19,14 +19,19 @@ open Path
 open Types
 open Btype
 
+let find_ident name =
+  Melange_wrapper.Predef.builtin_idents
+  |> List.find (fun (s, _) -> s = name)
+  |> snd
+
 let builtin_idents = ref []
 
 let wrap create s =
   let id = create s in
   builtin_idents := (s, id) :: !builtin_idents;
   id
 
-let ident_create = wrap Ident.create_predef
+let ident_create = wrap find_ident
```

Comment:
That's okay because it will fail fast and all of it's idents are already created.